.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ node-template: &node-template
           JEST_JUNIT_OUTPUT: "reports/junit/js-test-results.xml"
     - run: |
         if [[ $CIRCLE_JOB == current-release ]]; then
-          npm audit
+          ./scripts/better-audit.sh
```

Comment:
Unrelated to the purpose of the PR, but necessary in order for the build to pass. The problem is that `npm audit` will fail if there are any vulnerabilities at all, even if they're only in devDependencies, and currently some transitive dependencies of the dev tools simply don't have patches available. The `better-audit` script is what we've been using in our other JS projects to 1. allow the build to pass as long as none of the _runtime_ dependencies have vulnerabilities, and 2. make the output more readable than what you normally get from `npm audit`.